npdarrington/instaquizzes
# Nd save route

### What’s this PR do?  
- Allows a user to navigate to the save route and back to the home page
- Restructures what was previously inside of App to control the game inside of a Route render.
- Adds in a catch for router to send someone back to the home page if a match route is not found.
- Installs @types/react-router-dom that was previously missed at initial setup.
- Creates the SavedQuizzes component to display on the save route.
 
### Where should the reviewer start?  
- git pull this branch.
- `npm install` to install new dependency needed for routing with Typescript.
 
### How should this be manually tested?  
- `npm start` -> localhost:3000
- Click on the 'View Saved Quizzes' link to go to the save route.
- Click on the 'Return to InstaQuiz Game' link to go to the home route.
- Try to manually type in a route that doesn't exist to get sent back to home route.
- Play through a couple games to ensure that everything still works as intended.
 
### Any background context you want to provide?  
- This branch only got the save route set up. Displaying for the saved route will be the next branch.
- I believe a new Component should be created to take all of the code that is directly rendering on the home route and make it more modular.
 
### What are the relevant tickets?  
- closes #42 
- closes #43
